sklearn/linear_model/_base.py

```diff
@@ -419,6 +412,27 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+    normalize : bool, default=False
+        This parameter is ignored when ``fit_intercept`` is set to False.
+        If True, the regressors X will be normalized before regression by
+        subtracting the mean and dividing by the l2-norm.
+        If you wish to standardize, please use
+        :class:`sklearn.preprocessing.StandardScaler` before calling ``fit`` on
+        an estimator with ``normalize=False``.
```

Comment:
```suggestion
        :class:`~sklearn.preprocessing.StandardScaler` before calling ``fit``.
```

Reply:
This one was resolved by an update to upstream.